README.md

```diff
@@ -1,2 +1,43 @@
+[![Build Status](https://travis-ci.org/coniks-sys/libmerkleprefixtree-go.svg?branch=merkle-tree)](https://travis-ci.org/coniks-sys/libmerkleprefixtree-go)
+
 # libmerkleprefixtree-go
 A Merkle prefix tree implementation in Golang
+
+This library currently uses the `SHAKE128` ShakeHash with output size of 32 bytes. 
+
+The signature scheme is `Ed25519` signature algorithm.
+
+### Usage
+Initiate the tree & history
+```
+m := merkletree.InitMerkleTree(nonce, salt, publicKey, privateKey)
```

Comment:
This would benefit from seeing `public, private, _ = crypto.GenerateKey()` and describing `salt` and `nonce`.